MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManager.vfr

```diff
@@ -51,6 +51,20 @@ formset
             step    = 0,
     endnumeric;
 
+    checkbox varid  = BmmData.QuietBoot,
+              questionid = FORM_QUIET_BOOT_ID,
+              prompt = STRING_TOKEN(STR_QUIET_BOOT_PROMPT),
+              help   = STRING_TOKEN(STR_QUIET_BOOT_HELP),
+              flags  = RESET_REQUIRED,
+    endcheckbox;

```

Comment:
Can we make it so that the Quiet Boot checkbox is checked and grayed out if Fast Boot is enabled? Since Fast Boot is essentially a superset of Quiet Boot

Reply:
I don't see a reason to do so. The same way we would have to gray out timeout. And if we go further, we would have to select quiet boot and set timeout to 0  in the menu when fast boot is enabled. Then you would have to remember the setting from before enabling fast boot and restore it upon disabling. The overcomplication is the main reason I didn't go that way.

Reply:
Fair enough